### ISSUE TITLE
Unable to navigate nested tables in JAWS

### ISSUE THREAD

**Issue by aardrian** [FreedomScientific/standards-support#791]
### Summary

Table navigation commands (`Ctrl` + `Alt` + arrows) do not work in HTML tables nested within other HTML tables.

1. Using JAWS 2024 and Chrome, go to any of the following:
    * https://cdpn.io/aardrian/debug/RYQVeG
    * https://cdpn.io/aardrian/debug/EdvbWz
    * https://www.w3schools.com/howto/howto_html_table_nested.asp
2. In the first and second URL, expand any disclosure to reveal nested tables.
3. Press `T` to navigate to the parent table (the W3 Schools parent table is malformed and will not navigate).
4. Press `T` to navigate to the nested table (JAWS will announce correct column & row count and the nesting level).
5. Press `Ctrl` + `Alt` + arrow (any direction) and observe the announcement.

### Expected result

JAWS will move me into another cell, if there is one, based on the direction I have chosen.

### Actual result

For any cell, JAWS announces:
* "end of row" (for `Ctrl` + `Alt` + `→`),
* "beginning of row" (for `Ctrl` + `Alt` + `←`),
* "bottom of column" (for `Ctrl` + `Alt` + `↓`),
* "top of column" (for `Ctrl` + `Alt` + `↑`).

Interestingly, if I use the virtual cursor to move forward through the DOM, in a couple cases I could navigate to one or two more cells that I had visited, though I could not reproduce this consistently.

### Examples

* https://cdpn.io/aardrian/debug/RYQVeG
* https://cdpn.io/aardrian/debug/EdvbWz
* https://www.w3schools.com/howto/howto_html_table_nested.asp

The W3 Schools tables are malformed (no headers, no captions). The parent table has too few cells for it to be recognized as a table according to JAWS or Chrome (Chrome reports it as a `LayoutTable` while the nested table has the expected `table` role). This may be useful for debugging since the HTML seems to matter more than the exposed role.

### Other nuggets

The other two examples are valid HTML table structures. These two table example structures are used in the wild with at least two financial institutions off the top of my head (clients).

Works fine with JAWS 2024 / Firefox 120.

Found by Darrell Hilicker (https://mastodon.online/@darrell73/111491899590737429), confirmed by me and Jeff Bishop (https://mastodon.online/@JeffBishop/111494848612952976).

#### JAWS version and build number

* JAWS 2024.2310.70
* This works fine in JAWS 2023.
 
#### Operating System and version

Windows 11 23H2

#### Browser and version:

* Chrome 119
* Edge 119

**Comment by BrettLewisVispero** [comment]
This should be fixed in the December JAWS 2024 release. Thanks for reporting it.
